src/common/components/play/GameMode.tsx

```diff
@@ -68,18 +70,37 @@ export default class GameMode extends React.Component<GameModeProps> {
         )}
         <div
           style={{
-            display: 'flex',
             justifyContent: 'center',
             alignItems: 'center',
-            height: (compact || imagePath) ? 'auto' : 250,
-            textAlign: 'center',
-            fontSize: 24,
-            fontFamily: 'Carter One',
-            color: '#f44336',
-            WebkitTextStroke: '1px black'
+            width: compact ? '100%' : 'calc(100% - 220px)',
```

Comment:
`compact` is bigger than not?

Reply:
Yeah, it's because there are multiple GameModes per row when they're compact